### PR TITLE
Project Summary & Logo

### PR DESCRIPTION
**Pull request by leonardoz15**
Using the GatorGrader repository as an example, a project summary for GatorGrouper should give a complete description of what GatorGroup is and what makes it special. As per issue [114](https://github.com/GatorEducator/gatorgrouper/issues/114) in the main GatorGrader repo. This also includes a project logo at the top of the README.md.

### CODE REVIEW

**Comment by leonardoz15** [comment]
If we have any other features to add that I didn't include, please add them so we can create a PR in the main repo and collect more feedback.

File: README.md [hunk 1]
```diff
@@ -1,10 +1,22 @@
...
-GatorGrouper is a Python 3 program that assigns a list of students to groups of
-a specified size. The output of this program could then be communicated to the
-students in a specific class. Then, if a course instructor is using [GitHub
-Classroom](https://classroom.github.com/), you can ask the students in your
-class to create and join their assigned group.
+![logo](gatorgrouper_logo.svg "GatorGrouper")
+
+Designed for use with [GitHub](https://github.com/), [GitHub Classroom](https://classroom.github.com/),
+and [Travis CI](https://travis-ci.com/), GatorGrouper is an online group creation
+tool to help facilitate random group assignment in an educational setting.
```
Comment (quigley-c) [inline README.md]: not exactly random, as your next line points out

File: README.md [hunk 2]
```diff
@@ -1,10 +1,22 @@
...
+While other group management tools create random groups, GatorGrouper
+supports random and round robbing style grouping with the option to specify
```
Comment (quigley-c) [inline README.md]: round robin*

**Comment by leonardoz15** [comment]
I think its ready for merging into GatorGrouper, that way we can see what all the technical leads and Kapfhammer think.

**Comment by quigley-c** [comment]
Go ahead any add this to the main gatorgrouper repo

**Pull request closed without merging**